README.md

```diff
@@ -18,39 +18,30 @@ Originally developed by [Carlos Tadeu Panato Junior](https://github.com/cpanato/
 
 ### Step 1: Configure plugin in Mattermost
 
-1. Go to **System Console > Plugins > AWS SNS**:
+1. Go to **System Console > Plugins > AWS SNS**.
 
-  1. Set the channel to send notifications to, specified in the format `teamname,channelname`. If the specified channel does not exist, the plugin will create the channel for you. 
-      - Note: Must be the team and channel handle used in the URL. For example, in the following URL, set the value to `myteam,mychannel`: https://example.com/myteam/channels/mychannel
+  1. Set the channel to send notifications to, specified in the format `teamname,channelname`. If the specified channel does not exist, the plugin will create the channel for you.
+      - Note: Must be the team and channel handle used in the URL. For example, in the following URL, set the value to `myteam,mychannel`: https://example.com/myteam/channels/mychannel.
 
-  2. Set authorized users who can accept AWS SNS subscriptions. Must be a comma-separated list of user ID's.
+  2. Set authorized users who can accept AWS SNS subscriptions. Must be a comma-separated list of user IDs.
       - Tip: Use the [mattermost user search](https://mattermost.com/pl/cli-mattermost-user-search) CLI command to determine a user ID.
-
   3. Set the username that this integration is attached to.
   4. Generate a token used for an AWS SNS subscription. Copy this value as you will use it in a later step.
 
-2. Go to **System Console > Plugins > Management** and click **Enable** to enable the AWS SNS plugin.
+2. Go to **System Console > Plugins > Management** and select **Enable** to enable the AWS SNS plugin.
 
-Note: If you are running Mattermost v5.11 or earlier, you must first go to the [releases page of this GitHub repository](https://github.com/mattermost/mattermost-plugin-aws-SNS/releases), download the latest release, and upload it to your Mattermost instance [following this documentation](https://docs.mattermost.com/administration/plugins.html#plugin-uploads).
+**Note:** If you're running Mattermost v5.11 or earlier, you must first go to the [releases page of this GitHub repository](https://github.com/mattermost/mattermost-plugin-aws-SNS/releases), download the latest release, and upload it to your Mattermost instance [following this documentation](https://docs.mattermost.com/administration/plugins.html#plugin-uploads).
```

Comment:
Is this still valid? We've removed similar content from other plugin docs such as the GitHub docs. Added to this, should we mention self-hosted vs. cloud and differences between them?

Reply:
I don't think there are any differences between cloud and on-prem for this plugin

Reply:
Awesome. I'm happy to remove it pending @hanzei's feedback.

Reply:
I'm fine with removing this :+1: